03 - Showcases & Templates/Vaults/OB_Template.md

```diff
@@ -3,11 +3,12 @@ aliases:
 - 
 tags:
 - vault-kit
+- OB_Template
 publish: true
 ---
 
 # OB_Template
-Author: [[Zektor]]
+Author: [Hugo Sanots (Zektor)](Zektor)
```

Comment:
Spelling of your last name is off here, too.

Reply:
holly moly, i will not try to pull anything that late in the night again